files/default/upgrade-controller.sh

```diff
@@ -65,14 +49,10 @@ ceilometer-upgrade --skip-gnocchi-resource-types --config-file /etc/ceilometer/c
 set -e
 
 # Upgrade nova
-crudini --set /etc/nova/nova.conf upgrade_levels compute newton
+crudini --set /etc/nova/nova.conf upgrade_levels compute pike
```

Comment:
I think during the last upgrade I forgot to change this but I'll double check.

Reply:
Did you confirm that this should be Pike? The way the docs laid things out made it seem like the value here should be the lowest release on the cluster at any given time.

From the docs:

>[upgrade_levels] compute (String) Set a version cap for messages sent to compute services. Set this option to “auto” if you want to let the compute RPC module automatically determine what version to use based on the service versions in the deployment. Otherwise, you can set this to a specific version to pin this service to messages at a particular level. All services of a single type (i.e. compute) should be configured to use the same version, and it should be set to the minimum commonly-supported version of all those services in the deployment.

Reply:
Seems like I should change this to be ``auto`` then so I don't need to worry about this as much. Thanks for catching that!